src/solvers/dgmulti/dg.jl

```diff
@@ -260,7 +263,10 @@ function max_dt(u, t, mesh::DGMultiMesh,
   # `polydeg+1`. This is because `nnodes(dg)` returns the total number of
   # multi-dimensional nodes for DGMulti solver types, while `nnodes(dg)` returns
   # the number of 1D nodes for `DGSEM` solvers.
-  polydeg = rd.N
+  # For tensor-product elements the maximum of the polynomial degrees is used to ensure
```

Comment:
I'd specify for `TensorProductWedge` approximation types, since we don't yet support multiple polynomial degrees for more than just this one type.